docs/deployment.rst

```diff
@@ -1,15 +1,22 @@
+.. _deployment:
```

Comment:
I think this link is already created by the heading name below? (I forget how exactly rst does links)